pkg/providers/azure.go

```diff
@@ -320,15 +312,22 @@ func (sc *SetUpCmd) createFederatedCredentials() error {
 
 }
 
-func (sc *SetUpCmd) getAppObjectId(ctx context.Context) error {
+func (sc *SetUpCmd) getAppObjectId() error {
 	log.Debug("Fetching Azure application object ID")
 
-	appID, err := sc.AzClient.GraphClient.GetApplicationObjectId(ctx, sc.appId)
+	getObjectIdCmd := exec.Command("az", "ad", "app", "show", "--only-show-errors", "--id", sc.appId, "--query", "id")
```

Comment:
☕️ thanks for this, I am just curious why this is getting reverted? Using sdk for anything is better than using exec.command?

Reply:
sorry for the confusion, I have updated in the description let me know if you have any other questions.